docker/Dockerfile.debian

```diff
@@ -0,0 +1,40 @@
+# Use a build argument to specify the Rust builder image version
+ARG RUST_BUILDER_IMAGE=rust:1.67.0-slim-buster
+
+# Build Lemmy
+FROM $RUST_BUILDER_IMAGE as builder
+
+# Install build dependencies
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends libssl-dev pkg-config libpq-dev git && \
+    rm -rf /var/lib/apt/lists/*
+
+# Set the working directory to /app and copy the source code
+WORKDIR /app
+COPY . .
+
+# Set the version number
+RUN echo "pub const VERSION: &str = \"$(git describe --tag)\";" > "crates/utils/src/version.rs"
+
+# Release mode build
+RUN cargo build --release
+RUN cp ./target/release/lemmy_server /app/lemmy_server
+
+# The Debian runner
+FROM debian:buster-slim as lemmy
+
+# Install libpq for Postgres and other necessary packages
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends postgresql-client libc6 libssl1.1 ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+
+# Create a new user and group
+RUN addgroup --gid 1000 lemmy && \
+    useradd --no-create-home --shell /bin/sh --uid 1000 --gid 1000 lemmy
+
+# Copy resources and set permissions
+COPY --chmod=755 --from=builder /app/lemmy_server /app/lemmy
+
+USER lemmy
+EXPOSE 8536
+CMD ["/app/lemmy"]
```

Comment:
Is there any reason for adding a Debian dockerfile? I dont think we want to maintain another one.

Reply:
I just did not throw away the arm one ... and as it's not specific to arm anymore ... i did rename it to debian... as that's the base

I'm total fine with just deleting it

Reply:
@dessalines ☝️

Reply:
Ya I agree with deleting it. The less dockerfiles to maintain, the better.